flake.nix

```diff
@@ -100,6 +100,12 @@
           check-examples = checks.examples;
           check-readme-coherency = checks.readme-coherency;
 
+          coq-coverage-example = pkgs.callPackage ./examples/coverage {
+            inherit (packages) hax;
+            inherit (pkgs) coqPackages;
+            inherit craneLib;
+          };
```

Comment:
I think we should have the `coq-coverage-example` included in the `checks.examples` derivation, don't you think?
I don't see why this one example should be exposed specifically here

Reply:
Yes, there should not be a big difference from the other examples. My only worry is that it becomes quite a large job, if you want to run all of them in a single runner.